api-js/src/queries/domains/find-domains-by-org.js

```diff
@@ -14,7 +14,60 @@ const findDomainsByOrg = {
     },
     ...connectionArgs,
   },
```

Comment:
You can add a `deprecated:true` here I think to inform the front end guys that there's a different way to do this